### PR TITLE
Bug fix/miscellaneous improvements

### PR DESCRIPTION
**Pull request by chidozieononiwu**
- Fix dotnet branding. https://github.com/Azure/azure-sdk-tools/issues/3487
DotnetBranding
- Add button for finding first comment. https://github.com/Azure/azure-sdk-tools/issues/4085
First Comment Navigation
- Independent scrollbars between left and right review pane. https://github.com/Azure/azure-sdk-tools/issues/3639, https://github.com/Azure/azure-sdk-tools/issues/3410
- Improve Show Documentation Button. https://github.com/Azure/azure-sdk-tools/issues/4195
ShowDocsbutton

### CODE REVIEW

File: src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml [hunk 1]
```diff
@@ -71,15 +71,23 @@
...
+                @if(userPreference.HideLineNumbers == true)
                 {
                     var lineNumberClass = RemoveMultipleSpaces($"line-number d-none {lineClass}");
                     <td class="@lineNumberClass"><span>@Model.LineNumber</span></td>
                 }
                 else
                 {
-                    var lineNumberClass = RemoveMultipleSpaces($"line-number {lineClass}");
-                    <td class="@lineNumberClass"><span>@Model.LineNumber</span></td>
+                    if (Model.CodeLine.IsDocumentation)
```
Comment (praveenkuttappan) [inline src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml:81]: nit: This can be simplified to have same code at line 83 and 88 only once.

**Comment by praveenkuttappan** [comment]
Is there any issue to link to this PR?

**Comment by JonathanGiles** [comment]
Regarding the changes to the expand / collapse doc feature, just an FYI that @Vailorium is currently working in this area too to fix some issues (see #4052). There will potentially be some merge conflicts here.

**Comment by JonathanGiles** [comment]
I wonder if we should try to connect the vertical line with the box, to make it super clear which box is associated with which ref doc - otherwise I worry that if we expand two comments next to each other, it might be less clear which box is related to which ref doc:

image

**Comment by chidozieononiwu** [comment]
> I wonder if we should try to connect the vertical line with the box, to make it super clear which box is associated with which ref doc - otherwise I worry that if we expand two comments next to each other, it might be less clear which box is related to which ref doc:
> 
> image

I resolved this.

**Pull request merged**